netpyne_ui/netpyne_geppetto.py

```diff
@@ -135,32 +135,43 @@ def rename(self, path, oldValue,newValue):
                 newModel = model.replace(oldValue,newValue)
                 GeppettoJupyterGUISync.synched_models[newModel]=synched_component
 
+    def getPlotSettings(self, plot):
+        try:
```

Comment:
Instead of using a try/catch can we maybe do sth a bit more conservative? Maybe sth along these lines...
`if simConfig.analysis and plot in simConfig.analysis`